pkgs/top-level/haskell-ng.nix

```diff
@@ -39,6 +39,10 @@ rec {
     });
     ghc = compiler.ghc784;
 
+    ghcjs = packages.ghc7101.callPackage ../development/compilers/ghcjs {
+      ghc = compiler.ghc7101;
+      gmp = "${pkgs.gmp.override { withStatic = true; }}";
```

Comment:
I inherited that from @ryantrinkle's work. A lot of his work has involved getting things working for Darwin, and it could be that this is only necessary on that platform. I'd like to keep this PR focused to getting things working on Linux, though. I'll see if I can remove that for now.